airflow/providers/amazon/aws/hooks/sagemaker.py

```diff
@@ -1300,3 +1300,13 @@ def create_auto_ml_job(
             if "BestCandidate" in res:
                 return res["BestCandidate"]
         return None
+
+    async def describe_training_job_async(self, job_name: str) -> dict[str, Any]:
```

Comment:
Where does this get used?

Reply:
I refined the code a bit and used it in the trigger. Could you please take a look? Thanks!